include/signalrclient/connection.h

```diff
@@ -19,8 +19,8 @@ namespace signalr
 
     class connection
     {
-        // TODO: consider making public or remove?
-        typedef std::function<void(const utility::string_t&)> message_received;
+    public:
+        typedef std::function<void __cdecl (const utility::string_t&)> message_received_handler;
 
     public:

```

Comment:
I wanted to have `typedef`s in their own section but it does not really matter too much. Can remove.